src/modules/reserve/getReserverations.js

```diff
@@ -0,0 +1,7 @@
+const getReserverations = async (movieId) => {
+  const request = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/sUHYcv06blVrKgUc2kUH/reservations?item_id=${movieId}`);
+  const response = await request.json();
+  return response;
+};
+
+export default getReserverations;
```

Comment:
- [ ] Good Job for Implementing some of the point the previous reviewer suggest just like possible to reserve any of the movies that we have listed, As the wire frame suggested the pop-up  centered on the page .
- [ ] But Kindly I suggest you to implemented the following point which should be addressed.
- When a user clicks on the "Reserve" button the screen  updated a result with the new reservation we make but it replace the First one with the new reservation I make Just like the video below.
- I suggest adding a reservation counter on the pop-up window, and the value should be incremented accordingly. when we add a new reservation the reservation count should be incremented and the screen is populated with the new count. Remember, The counter should be implemented as a separate module.

Please check this [video](https://drive.google.com/file/d/1a5HZ77WRuXSJoCYtksimVOIpzSCeuDEN/view?usp=share_link)